CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.1.1-dev
```

Comment:
It looks like this should be rolled into the 1.1.0-dev changelog/version?

Reply:
What do you mean? I bumped the version because reportLines requires vm_service 7.3.0. It's possible that users have other versioning constraints keeping them on older versions of vm_service, so this needs to be a separate version of package:coverage so that pub can keep them on 1.1.0 if necessary. I ran into this issue with dwds.

Reply:
`1.1.0` has not been published so we can just merge with the section below, that is `1.1.0-dev`

Reply:
Ah ok, I suppose that's what the -dev means. Done